json-server/config/db.js

```diff
@@ -2012,6 +2012,11 @@ module.exports = function () {
                 "isMainContact": false,
                 "isNamedDeputy": false
             },
+
+            {
+                "id": "deleteContact"
```

Comment:
Kate's working on a better way of doing this in the firm hub, would be nice to apply it here too